File: C++11/백준/Bronze/3449. 해밍 거리/해밍 거리.cc
#include <iostream>
#include <string>
using namespace std;

int main(void){
    int a;
    cin >> a;
    for (int i=0;i<a;i++){
        string s1,s2;
        int cnt = 0;
        cin >> s1 >> s2;
        for (int j=0;j<s1.size();j++){
            if (s1[j]!=s2[j]){
                cnt++;
            }
        }
        cout << "Hamming distance is " <<cnt <<"." << '\n';
    }
    return 0;
}

File: Java/백준/Bronze/3449. 해밍 거리/해밍 거리.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int t = in.nextInt();
        for (int i=0;i<t;i++){
            String s1 = in.next();
            String s2 = in.next();
            int cnt = 0;
            for (int j=0;j<s1.length();j++){
                if(s1.charAt(j)!=s2.charAt(j)){
                    cnt++;
                }
            }
            System.out.printf("Hamming distance is %d.\n",cnt);
        }
    }
}